docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,632 @@
+# TAD Reporting Standard
+
+This document describes the Transparancy of Algorithmic Decision Making (TAD) reporting standard.
+
+## Introduction
+
+The standard almost [^1] [^2] extends the [Hugging Face model card metadata specification](https://github.com/huggingface/hub-docs/blob/main/modelcard.md?plain=1)
+to allow for:
+
+1. More finegrained information on performance metrics, by extending the `metrics_field` from the Hugging
+Face metadata specification.
+2. Capturing additional *measurements* on fairness and bias, which can be partitioned into bar plot like
+measurements (such as mean absolute SHAP values) and graph plot like measurements (such as partial dependence). This is achieved
+by defining an new field `measurements`.
+3. Capturing *assessments* (such as IAMA). This is achieved by defining a new field `assessments`.
+
+This standard does not contain all fields present in the Hugging Face metadata specification. The fields that
+are optional in the HuggingFace specification AND are specific to the HugginFace interface are ommited.
+
+Following Hugging Face, this proposed standard will be written in yaml.
+
+## Intended usage
+
+The reporting standard defined in this document is intented to be given as a preamble in a model card, similar to
+the Hugging Face metadata.
+
+## Specification of the standard
+
+The standard will be written in yaml. An example yaml is given in the next section. The standard contains the
+following information.
+
+1. `algorithm-registers` (OPTIONAL, list[string]). If this algorithm is registered in an algorithm register,
+this field should contain a URI to the corresponding record in the register. There can be multiple registers.
```

Comment:
```suggestion
this field should contain a URI to the corresponding record in the register. There can be multiple registers. A URI could look like `https://huggingface.co/org/modelid`.
```

Reply:
Is there a reason your example URI is huggingface and not from algoritmeregister?

Reply:
Maybe include both?